src/Oci8/Query/Grammars/OracleGrammar.php

```diff
@@ -203,7 +203,7 @@ protected function wrapValue($value)
             return $value;
         }
 
-        $value = $this->isReserved($value) ? Str::lower($value) : Str::upper($value);
+        $value = Str::upper($value);
```

Comment:
Ahh, yeah, that makes sense. And we can certainly wait for that change to be there until a major version. I'll add the migration change to my PR and retarget.